apps/sms/views/shared/test/unit/settings_test.js

```diff
@@ -222,9 +235,9 @@ suite('Settings >', function() {
       navigator.mozMobileConnections = [{ iccId: 'SIM 1' }, { iccId: null }];
       Settings.init();
 
-      // Four calls for
-      // maxConcatenatedMessages/mmsSizeLimitation/mmsServiceId/smsServiceId
-      sinon.assert.callCount(navigator.mozSettings.createLock, 4);
+      // Five calls for maxConcatenatedMessages/mmsSizeLimitation
+      // sendReadReport/mmsServiceId/smsServiceId/
+      sinon.assert.callCount(navigator.mozSettings.createLock, 5);
```

Comment:
Yeah, They are implied by other checks. In my POV, it's better to have it, as it give clear understanding of test that ,what n all and where these locks created.